src/components/sidebar/sidebar.css

```diff
@@ -12,6 +12,8 @@
 }
 
 .sidebar-contents-wrapper {
+  /* Allow absolute positioning that is independent of the scrolling. */
+  position: relative;
```

Comment:
Because I was curious, I tried other things locally, like adding `position: relative` to a closer element, but the toggles would just disappear. I don't exactly understand why, but well...